unlock-app/src/components/interface/keychain/Key.tsx

```diff
@@ -54,52 +69,57 @@ const KeyBox = ({
   const [isCopied, setCopied] = useClipboard(lock.address, {
     successDuration: 2000,
   })
+
+  // Replace with actual call to backend
+  const nextRenewal = '9 PM : 12 March 2022'
+  const userBalance = '12 LINK'
+  const approvedTime = '12 days'
+  const type = 'crypto'
```

Comment:
I am not sure we have an easy way to distinguish things between crypto/card/grants... etc so Probably better to skip!

Reply:
The subscription endpoint tries to guess based on what information we have.